include/aws/common/byte_buf.h

```diff
@@ -368,6 +368,19 @@ int aws_byte_buf_append_with_lookup(
 AWS_COMMON_API
 int aws_byte_buf_append_dynamic(struct aws_byte_buf *to, const struct aws_byte_cursor *from);
 
+/**
+ * Copies from to to. If to is too small, the buffer will be grown appropriately and
```

Comment:
I suggest we rename all these args to `src` and `dest`

Reply:
It's worth considering, but for now I'm just going to put quote ticks around the names for clarity.